src/actions.rs

```diff
@@ -0,0 +1,497 @@
+//! Actions functionality for WebDriver.
+use crate::elements::Element;
+#[cfg(doc)]
+use crate::keys::Key;
+use crate::Client;
+use std::fmt::Debug;
+use std::time::Duration;
+use webdriver::actions as WDActions;
+
+/// An action not associated with an input device (i.e. pause).
```

Comment:
```suggestion
/// An action not associated with an input device (e.g. pause).
```
`pause` is the only example we have currently, but I don't think we know that it's the only _possible_ null action for all future.